src/test/suite/grammar.test.ts

```diff
@@ -0,0 +1,78 @@
+import * as assert from 'assert';
+import * as vscode from 'vscode';
+
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const vsctm = require('vscode-textmate');
+const oniguruma = require('vscode-oniguruma');
```

Comment:
I don't think there's a functional difference in this case, but we can make these more consistent by only using one form of importing. In any case, it seems the Typescript way is to use `import fs = require('fs')` instead of `const`. For example, you don't get code completion or type inference with the latter, at least in VS Code.

I believe that `import ... from` is in a more recent addition to the standard and is supposed to be preferable, although I'm not sure. It also lets you import individual functions etc. rather than the whole module. It doesn't work for some "old-style" packages that export a single function or value, but it should be fine with all of these. See [some more information](https://www.typescriptlang.org/docs/handbook/modules.html#import).

Reply:
I used `import * as X from 'Y';`, seems to work for all packages and it's more consistent with the above lines.